src/Templates.json

```diff
@@ -0,0 +1,100 @@
+[{"id":1,"title":"Wiberg Super Cure","price":"$5.41"},
```

Comment:
you need to store this file in some assets folder or on the place where you have redux